platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -13,6 +13,27 @@
 // limitations under the License.
 
 [[upgrading_version_8]]
+= Upgrading your build from Gradle 8.x to Gradle 9.0
+
+[[unix_file_permissions_removed]]
```

Comment:
Can you move this entry to line 63?

Reply:
Applied all your suggestions. Thank you.